kinto/core/resource/viewset.py

```diff
@@ -157,8 +156,6 @@ def get_record_schema(self, resource_cls, method):
         else:
             resource_schema = resource_cls.schema
             if hasattr(resource_cls, "mapping"):
-                message = "Resource `mapping` is deprecated, use `schema`"
-                warnings.warn(message, DeprecationWarning)
                 resource_schema = resource_cls.mapping.__class__
```

Comment:
We should remove the deprecated fallback too, no just remove the warning message :) 
In other words, we should stop supporting the `.mapping` attribute on the resource class